src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+    public static final String LINESEPARATOR = "____________________________________________________________\n";
+    public static final int MAX_TASKS = 100;
```

Comment:
Good job declaring a magic number as a constant and following the coding standard!